# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gst-plugins-good/compile-fix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gst-plugins-good-0.10.6-orig/ext/taglib/gstid3v2mux.cc gst-plugins-good-0.10.6/ext/taglib/gstid3v2mux.cc
--- gst-plugins-good-0.10.6-orig/ext/taglib/gstid3v2mux.cc	2008-01-16 17:16:12.000000000 +0000
+++ gst-plugins-good-0.10.6/ext/taglib/gstid3v2mux.cc	2008-01-16 17:18:32.000000000 +0000
@@ -544,7 +544,9 @@
   GST_TAG_MUSICBRAINZ_ALBUMARTISTID, add_musicbrainz_tag, "\002"}, {
   GST_TAG_MUSICBRAINZ_TRMID, add_musicbrainz_tag, "\003"}, {
   GST_TAG_MUSICBRAINZ_TRACKID, add_unique_file_id_tag, ""}, {
-  GST_TAG_MUSICBRAINZ_SORTNAME, add_text_tag, "TSOP"}, {
+  GST_TAG_ARTIST_SORTNAME, add_text_tag, "TSOP"}, {
+  GST_TAG_ALBUM_SORTNAME, add_text_tag, "TSOA"}, {
+  GST_TAG_TITLE_SORTNAME, add_text_tag, "TSOT"}, {
   GST_TAG_TRACK_NUMBER, add_count_or_num_tag, "TRCK"}, {
   GST_TAG_TRACK_COUNT, add_count_or_num_tag, "TRCK"}, {
   GST_TAG_ALBUM_VOLUME_NUMBER, add_count_or_num_tag, "TPOS"}, {
